libraries/adaptive-expressions/tests/expressionParser.test.js

```diff
@@ -508,6 +508,8 @@ const testCases = [
             ['date(timestamp)', '3/15/2018'],//Default. TODO
             ['year(timestamp)', 2018],
             ['length(utcNow())', 24],
+            ['formatDateTime(notISOTimestamp, \'dd/MM/yyy\')', '15/03/2018'],
```

Comment:
Do you think a test case for `'dd/MM/yy'` is necessary? The changes in datetimeFormatConverter are trivial so I don't believe another test case for that is required, but let's be explicit since this is a customer ask.

[Relevant comment](https://github.com/microsoft/botbuilder-js/issues/2864#issuecomment-704507726)

Reply:
add a test cases in addDays function